tree/ntuple/doc/BinaryFormatSpecification.md

```diff
@@ -684,6 +685,64 @@ The entry range allows for finding the right page list for random access request
 The number of clusters information allows for using consistent cluster IDs
 even if cluster groups are accessed non-sequentially.
 
+#### Linked Attribute Sets
+
+An RNTuple may have zero or more linked Attribute Sets, containing metadata.
+Each Attribute Set is stored on disk as an RNTuple and the Anchor of each RNTuple is linked to by the main
+RNTuple's footer.
+
+An Attribute Set RNTuple has a number of restrictions compared to a regular RNTuple:
+
+1. it cannot have linked Attribute RNTuples itself;
+2. the Alias columns sections, both in its header and footer, must be empty (i.e. none of the Attribute Set RNTuple's
+   Fields can be Projected Fields);
+3. none of its fields may have a structural role of 0x04 (i.e. it must not contain a ROOT streamer object);
+
+The payload of each list element is a Record Frame containing the following fields:
+
+- a 32-bit unsigned integer containing the _Attribute Schema Version_. This is split into a _Major_ (16 LSB) and a
+  _Minor_ (16 MSB) version. The Schema Version is described below;
+- a 32-bit unsigned integer containing the uncompressed size of the Anchor;
+- a Locator pointing to the Attribute RNTuple's Anchor;
+- a String containing the Attribute Set's name. All linked Attribute Sets must have a non-empty, distinct name.
+
+```
+ 0                   1                   2                   3
+ 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|     Schema Version Major     |     Schema Version Minor       |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|             Attribute Anchor Uncompressed Size                |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                   Attribute Anchor Locator                    |
+.                             ...                               .
+|                                                               |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                      Attribute Set Name                       |
+.                             ...                               .
+|                                                               |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
```

Comment:
For variable record frames, we otherwise use "followed by [...]" (the attribute anchor locator and the attribute set name)